device/vxlan/unicast.yaml

```diff
@@ -1,14 +1,10 @@
 components:
   schemas:
-    Vxlan.Unicast:
+    Vxlan.Tunnel.DestinationIPMode.UnicastBase:
       description: >-
-        Configuration of Unicast parameters for VxLAN.
+        DestinationIPMode Unicast Base
       type: object
       properties:
-        remote_vtep_address:
-          description: >-
-            Remote VXLAN Tunnel End Point address
-          $ref: '#/components/schemas/Remote.Vtep.Address'
         suppress_arp:
```

Comment:
Since this is the base object for both IPv4 and IPv6, it is correct to call it 'suppress_arp'?  In IPv6, the protocol is NDP (Neighbor Discovery protocol).  Maybe remove this parameter from the base object and move it into the derived objects where it can be named as 'suppress_arp' for IPv4 and 'suppress_ndp' for IPv6?

Also typo in the description : 'Supress' --> 'Suppress'

Reply:
'suppress_arp' is removed, if user populates 'arp_suppression_cache' the 'suppress_arp' will be set to true. As of now I could see support only for IPv4 even though the tunnel is IPv6. In future we can add 'nd_suppression_cache' if there is support needed and this won't be a breaking change.